### PR TITLE
Add es6 support

### PR DESCRIPTION
**Pull request by LionOps**
While this branch was in Testing, commits were made to test snitch. I rebased it to clean up the history.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var assign = require('lodash.assign');
+let assign = require('lodash.assign');
```
Comment (snitch) [inline src/index.js]: 'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var assign = require('lodash.assign');
-var ResultList = require('./result-list');
+let ResultList = require('./result-list');
```
Comment (snitch) [inline src/index.js]: 'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: src/index.js (hunk 1 above)
Comment (snitch) [inline src/index.js]: 'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var assign = require('lodash.assign');
+let assign = require('lodash.assign'); // sup snitch?
```
Comment (snitch) [inline src/index.js]: 'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by kahnvex** [comment]
:+1:

**Pull request merged**